cmd/snap/main.go

```diff
@@ -418,6 +418,9 @@ func main() {
 	// no magic /o\
 	if err := run(); err != nil {
 		fmt.Fprintf(Stderr, errorPrefix, err)
+		if client.IsRetryableError(err) {
+			os.Exit(10)
```

Comment:
Is the exit code significant?

Reply:
I would've gone with 9 (because it's http 409) :-)